build.sbt

```diff
@@ -13,25 +16,29 @@ val standardSettings = Seq[Setting[_]](
   riffRaffArtifactPublishPath := name.value
 )
 
+//Common project
 lazy val common = project
   .settings(LocalDynamoDB.settings)
   .settings(standardSettings: _*)
   .settings(
     resolvers ++= Seq(
       "Guardian GitHub Releases" at "http://guardian.github.com/maven/repo-releases",
-      "Guardian GitHub Snapshots" at "http://guardian.github.com/maven/repo-snapshots"
+      "Guardian GitHub Snapshots" at "http://guardian.github.com/maven/repo-snapshots",
+      "Guardian Platform Bintray" at "https://dl.bintray.com/guardian/platforms"
     ),
     libraryDependencies ++= Seq(
       json,
       ws,
       "com.microsoft.azure" % "azure-servicebus" % "0.7.0",
       "org.typelevel" %% "cats" % "0.7.0",
       "joda-time" % "joda-time" % "2.8.2",
-      "com.amazonaws" % "aws-java-sdk" % "1.9.31",
+      "com.amazonaws" % "aws-java-sdk" % "1.11.205",
+      "com.amazonaws" % "aws-java-sdk-dynamodb" % "1.11.205",
```

Comment:
Isn't dynamoDB part of the SDK just above?

Reply:
Yup you're right. That's what I need to do. Pay attention and take care . :(